### PR TITLE
added ability to create/query Jira webhooks

### PR DESCRIPTION
**Pull request by guymatz**
resolves https://github.com/pycontribs/jira/issues/272

### CODE REVIEW

**Comment by guymatz** [comment]
Anyone know what I need to do to fix this?

**Comment by ssbarnea** [comment]
@guymatz Please rebase your code in order to resolve conflicts and pass the tests. Once these are done I will finish reviewing it and merging. Otherwise I will close the PR as it would become too old.

**Comment by guymatz** [comment]
Great!  Thanks!!

**Comment by ssbarnea** [comment]
Run "python -m flake8" and assure this is passing. If needed you may have to install flake8 via "pip install flake8".

File: .travis.yml [hunk 1]
```diff
@@ -32,13 +32,15 @@ notifications:
 deploy:
 - provider: releases
   api_key:
-    secure: gr9iOcQjdoAyUAim6FWKzJI9MBaJo9XKfGQGu7wdPXUFhg80Rp6GLJsowP+aU94NjXM1UQlVHDAy627WtjBlLH2SvmVEIIr7+UKBopBYuXG5jJ1m3wOZE+4f1Pqe9bqFc1DxgucqE8qF0sC24fIbNM2ToeyYrxrS6RoL2gRrX2I=
+    secure: G19YtkGAX0aJ1oyd/7eRj1KYdsmAkjkfU2UISvsjh/68ec1+9qtPpN7BbkFYZYMjSx0BtS0SEEA7Vdl4F9DI9Zzqahbj7WzDLFe9/4aZKM/ztfKWR6CNAYaMazAKS5W7r9pPkBBDIIJ9zCqvV7FRzjewEpfTwFzwUdY+IpxEsAM=
```
Comment (ssbarnea) [inline .travis.yml]: Your change is not supported to touch the travis file, probably that's because you didn't rebase your code.

**Comment by guymatz** [comment]
I think I'm just going to start over.  Is that OK?

**Comment by ssbarnea** [comment]
sure, cancel and atart over if it is easier.

-- sorin

**Pull request closed without merging**